### PR TITLE
feat(Dropdown): add tabSelectsValue support

### LINKED ISSUE #1204: No tabSelectsValue prop in Dropdown
When `Dropdown` options are opened and tab is pressed - focused option becomes selected.
This behaviour can be changed with `tabSelectsValue` prop in `Select` component from `react-select` but we don't have access to it through Dropdown props.

Could you please expose `tabSelectsValue` to `Dropdown` props  so this behaviour could be customizable ?

Example with `tabSelectsValue={true}` or `tabSelectsValue={undefined}` (only possible behaviour for now) 

https://user-images.githubusercontent.com/108681710/230099119-0e0c6cf2-df70-4d8a-a373-be0245521c97.mp4

<br />

Example with `tabSelectsValue={false}`

https://user-images.githubusercontent.com/108681710/230099036-e9155335-6c44-44a1-b63f-87bb7bbe331a.mp4

### PR DESCRIPTION
**Pull request by naorpeled**
resolves #1204

### CODE REVIEW

**Comment by shlomitc** [comment]
This PR has been stale for a while so I'm closing it. Thanks

**Comment by naorpeled** [comment]
Hey @shlomitc,
It's been stale due to it not being reviewed.
Imo it was ready for review, feel free to review it if interested ❤️

**Comment by YossiSaadi** [comment]
@naorpeled there are some conflicts due to stale (from our side)
once will be solved we'll be able to merge :)

**Comment by naorpeled** [comment]
> @naorpeled there are some conflicts due to stale (from our side) once will be solved we'll be able to merge :)

Awesome, will rebase soon, thanks!!

**Comment by naorpeled** [comment]
@YossiSaadi I rebased but need to do some testing,
having trouble with running the storybook for some reason.
Will be able to take a further look later/tomorrow.

**Comment by YossiSaadi** [comment]
> @YossiSaadi I rebased but need to do some testing, having trouble with running the storybook for some reason. Will be able to take a further look later/tomorrow.

thank you, looking forward :) ping me here

File: src/components/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -223,7 +224,7 @@ const Dropdown = forwardRef(
         insideOverflowWithTransformContainer,
         controlRef,
         tooltipContent,
-        popupsContainerSelector
+        popupsContainerSelector,
```
Comment (naorpeled) [inline src/components/Dropdown/Dropdown.jsx:227]:
```suggestion
        popupsContainerSelector
```

**Comment by naorpeled** [comment]
@YossiSaadi done testing, looks good now.

**Comment by YossiSaadi** [comment]
Thanks for your patience @naorpeled!
Merged it, much appreciated!

**Pull request merged**